projects/tuist/fixtures/ios_app_with_custom_workspace_rendering_markdown/Workspace.swift

```diff
@@ -0,0 +1,15 @@
+import ProjectDescription
+
+let workspace = Workspace(
```

Comment:
I would probably edit the `projects/tuist/fixtures/ios_app_with_custom_workspace/` instead of adding a whole fixture for this small feature